src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/FormattingSpan.cs

```diff
@@ -7,7 +7,15 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting
 {
     internal class FormattingSpan
     {
-        public FormattingSpan(TextSpan span, TextSpan blockSpan, FormattingSpanKind spanKind, FormattingBlockKind blockKind, int razorIndentationLevel, int htmlIndentationLevel, bool isInClassBody = false)
+        public FormattingSpan(
+            TextSpan span,
+            TextSpan blockSpan,
+            FormattingSpanKind spanKind,
+            FormattingBlockKind blockKind,
+            int razorIndentationLevel,
+            int htmlIndentationLevel,
+            bool isInClassBody = false,
+            int componentLambdaNestingLevel = 0)
```

Comment:
Hurts to capture this as part of the formatting span tech but I 100% understand why we have to.

Reply:
Yup 😢. This was the only quick way to do it without changing the whole world.